nuxeo-features/nuxeo-platform-relations/nuxeo-platform-relations-jena-plugin/src/test/java/org/nuxeo/ecm/platform/relations/jena/TestJenaGraph.java

```diff
@@ -108,6 +112,15 @@ protected void postSetUp() throws Exception {
         Collections.sort(statements);
     }
 
+    /**
+     * NXP-22534 This fix has to be discussed
+     */
+    @After
+    public void after() {
+        Framework.getRuntime().getComponentManager().reset();
```

Comment:
If it the same thing than WorkManager, we should definitively create a Feature to be able to restart runtime between tests.